src/components/BookItem.js

```diff
@@ -1,43 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/Books/books';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import { deleteBook } from '../redux/Books/books';
 
 const BookItem = ({ book }) => {
   const {
-    booktitle, id, author, category,
+    title, author, id, category,
   } = book;
   const dispatch = useDispatch();
 
-  return (
-    <li key={id}>
-      <p>
-        Book:
-        {' '}
-        { booktitle }
-      </p>
-      <p>
-        Author:
-        {' '}
-        { author }
-      </p>
+  const HandleRemove = () => {
+    dispatch(deleteBook(id));
+  };
+
+  const chapterVal = Math.round(Math.random() * 20);
 
-      <p>
-        Category:
-        {' '}
-        {category}
-      </p>
-      <button type="button" onClick={() => dispatch(removeBook(id))}>Remove</button>
+  return (
+    <li key={id} className="d_flex">
+      <div className="firsCol">
+        <p className="category">{category}</p>
+        <p className="title">{ title }</p>
+        <p className="Author" required>{ author }</p>
+        <div className="firsCol-btns">
+          <button type="button">Comment</button>
+          <button type="button" onClick={HandleRemove}>Remove</button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <ul className="secondCol d_flex">
+        <li>
+          <div style={{ width: 100, height: 100 }}>
```

Comment:
- [ ] For best practice, kindly create a variable for React inline style and pass it to React style e.g
``` javascript
  const styvar = {
     width: 100,
     height: 100
  }
```